man/ocitools-generate.1.md

```diff
@@ -47,6 +47,10 @@ compatible runtime like runC to run a container.
 **--cwd**=PATH
   Current working directory for the process
 
+**--cgroups-path**=""
```

Comment:
“cg” comes before “cw” in alphabetical order, so this entry should land between `--cap-drop` and `--cwd`.